berrylan.apparmor

```diff
@@ -0,0 +1,6 @@
+{
+    "policy_groups": [
+        "networking"
```

Comment:
I think networking isn't even required for this...

Reply:
Actually I've been wrong. It's required for the FAQ which are loaded from web server.